### PR TITLE
DD 10283: Change Country Dropdown to Radio Buttons - to test 

### PR DESCRIPTION
**Pull request by jasonfu9966**
only minor refactoring happened in this PR

### CODE REVIEW

**Review by TomElliottFlexion — APPROVED** [review]

File: web-client/src/views/StartCase/Country.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const Country = connect(
               className="usa-radio__label"
               data-testid="international-country-btn"
```
Comment (nechama-krigsman) [inline web-client/src/views/StartCase/Country.tsx]: Should the data-testid change here as well?

File: web-client/src/views/StartCase/Country.tsx [hunk 2]
```diff
@@ -84,7 +84,7 @@ export const Country = connect(
             />
             <label
               className="usa-radio__label"
-              data-testid="international-country-btn"
+              data-testid="domestic-country-btn"
```
Comment (nechama-krigsman) [inline web-client/src/views/StartCase/Country.tsx]: Is this testId correct?

File: web-client/src/views/StartCase/Country.tsx (hunk 1 above)
Comment (nechama-krigsman) [reply web-client/src/views/StartCase/Country.tsx]: Never mind...i was looking at old commit.

Comment (jasonfu9966) [reply web-client/src/views/StartCase/Country.tsx]: not a bad catch though!

File: web-client/src/views/StartCase/Country.tsx (hunk 2 above)
Comment (nechama-krigsman) [reply web-client/src/views/StartCase/Country.tsx]: same here!

**Review by nechama-krigsman — APPROVED** [review]

**Pull request merged**